docker/jenkins/Dockerfile.debian9-x86_64

```diff
@@ -0,0 +1,56 @@
+FROM debian:stretch
+
+RUN set -x \
+    && apt-get update -y
+
+RUN apt-get install -y \
+    ant \
+    libboost-all-dev \
+    bzip2 \
+    cmake \
+    fakeroot \
+    gcc \
+    git \
+    gstreamer1.0-plugins-base \
+    openjdk-8-jdk  \
+    libpam0g-dev \
+    libffi-dev \
+    uuid-dev \
+    make \
+    libssl1.0-dev \
+    libpango-1.0-0 \
+    r-base \
+    sudo \
+    wget \
+    libxml-commons-external-java \
+    mesa-common-dev \
+    zlib1g \
+    libattr1-dev \
+    libcap-dev \
+    libacl1-dev
+
+## run install-boost twice - boost exits 1 even though it has installed good enough for our uses.
+## https://github.com/rstudio/rstudio/blob/master/vagrant/provision-primary-user.sh#L12-L15
+COPY dependencies/common/install-boost /tmp/
+RUN bash /tmp/install-boost || bash /tmp/install-boost
+
+# install cmake
+COPY package/linux/install-dependencies /tmp/
+RUN bash /tmp/install-dependencies
+
+# install Qt SDK
+COPY dependencies/linux/install-qt-sdk /tmp/
+RUN mkdir -p /opt/RStudio-QtSDK && \
+    export QT_SDK_DIR=/opt/RStudio-QtSDK/Qt5.4.0 && \
```

Comment:
this should use our build of Qt 5.4.2 which supports gstreamer-1.0; see e.g. https://github.com/rstudio/rstudio/blob/master/docker/jenkins/Dockerfile.xenial-amd64#L49-L53